### ISSUE TITLE
[Desktop] Brave stats use wrong word for "saved" in German translation

### ISSUE THREAD

**Issue by dhaun** [brave/brave-browser#10512]
## Description 

tl;dr Brave uses the German word "gespeichert" for "saved" where it should instead use the German word "gespart".

The Brave statistics as displayed in a new window have a wrong translation for the word "saved" (as in so-and-so much times saved). That's because the English word can both mean "stored" (as in: saved to disk) and "held back" or "not used" (as in: saved time / bandwidth).

## Steps to Reproduce

   1. Have the Brave stats switched on
   2. Have the Browser / system language set to German
   3. Open a new window

## Actual result:

Says things like:

- 88.6 MB Bandbreite gespeichert
- 3 Minuten Zeit gespeichert

## Expected result:

Should say:

- 88.6 MB Bandbreite gespart
- 3 Minuten Zeit gespart

## Reproduces how often: 

Every time

## Version/Channel Information:

Brave | 1.10.97 Chromium: 83.0.4103.116 (Official Build) (64-bit)
-- | --
Revision | 8f0c18b4dca9b6699eb629be0f51810c24fb6428-refs/branch-heads/4103@{#716}
OS | macOS Version 10.15.5 (Build 19F101)

This is the current version at the time of this writing. Haven't tried any beta, dev, or nightly builds.

## Other Additional Information:

n/a

## Miscellaneous Information:
![wrong-german-translation-for-saved](https://user-images.githubusercontent.com/16507691/86040337-23d62280-ba44-11ea-9ef4-63e88003dc5e.png)

n/a

**Comment by mkarolin** [comment]
Fixed by https://github.com/brave/brave-core/pull/6024
